autogen/coding/local_commandline_code_executor.py

```diff
@@ -271,70 +283,3 @@ def _execute_code_dont_check_setup(self, code_blocks: List[CodeBlock]) -> Comman
     def restart(self) -> None:
         """(Experimental) Restart the code executor."""
         warnings.warn("Restarting local command line code executor is not supported. No action is taken.")
-
```

Comment:
Why should we remove this? 

cc @jackgerrits

Reply:
Added